tests/Behat/Content/Node/ProjectContentContext.php

```diff
@@ -84,6 +85,11 @@ public function alterProjectFields(BeforeParseEntityFieldsScope $scope): void {
           $scope->addFields($fields)->removeField($key);
           break;
 
+        case 'Project contact':
+          $fields = $this->getReferenceRevisionField($mapping[$key], 'oe_contact', $value);
+          $scope->addFields($fields)->removeField($key);
+          break;
+
```

Comment:
@yenyasinn We would need to create a behat test that tests that all this "API steps" are actually working. We can do this in a follow up, please create one, thank you!